src/components/overlays/pause.js

```diff
@@ -20,9 +20,20 @@ export class Pause extends Screen {
 
         const parentKey = this.context.parentScreens.slice(-1)[0].scene.key;
         const showReplay = Boolean(this.context.navigation[parentKey].routes.restart);
+        const showSelect = Boolean(this.context.navigation["pause"].routes.select);
 
-        const replayButton = showReplay ? ["pauseReplay"] : [];
+        const selectButton = ["levelSelect"];
+        const replayButton = ["pauseReplay"];
+        const replayAndSelectButton = ["levelSelect", "pauseReplay"];
 
-        this.setLayout(buttons.concat(replayButton));
+        if (showReplay && !showSelect) {
+            this.setLayout(buttons.concat(replayButton));
+        } else if (!showReplay && showSelect) {
+            this.setLayout(buttons.concat(selectButton));
+        } else if (showReplay && showSelect) {
+            this.setLayout(buttons.concat(replayAndSelectButton));
+        } else {
+            this.setLayout(buttons);
```

Comment:
this could defo be tidied up a bit, something like:

```
showReplay && buttons.concat(replayButton)
showSelect && buttons.concat(selectButton)
```

Reply:
You can't concat more than once it explodes

Reply:
oh right you need to a setter then
```
showReplay && (buttons = buttons.concat(replayButton))
showSelect && (buttons = buttons.concat(selectButton))
```

Reply:
I'm pretty sure it still doesn't work, there is something weird going on with buttons.concat

Reply:
🤔

Reply:
Hm this specific way does work, was doing some other concating and it broke, changed thanks